query/src/test/java/org/infinispan/query/dsl/embedded/impl/model/TheEntity.java

```diff
@@ -26,7 +26,7 @@ public TheEntity(String fieldX, TheEmbeddedEntity embeddedEntity) {
       this.embeddedEntity = embeddedEntity;
    }
 
-   @SortableField
+   @SortableField(forField = "theField")
```

Comment:
This change is only necessary because the previous parsing of `@SortableField` was buggy.

If you look at the history, we actually used to set the `forField` attribute in Infinispan 11 (because it was necessary), so we're back to normal.